Doc/c-api/tuple.rst

```diff
@@ -149,8 +156,8 @@ type.
 
 .. c:function:: int PyStructSequence_InitType2(PyTypeObject *type, PyStructSequence_Desc *desc)
 
-   The same as ``PyStructSequence_InitType``, but returns ``0`` on success and ``-1`` on
-   failure.
+   The same as ``PyStructSequence_InitType``, but returns ``0`` on success
```

Comment:
```suggestion
   The same as :c:func:`PyStructSequence_InitType`, but return ``0`` on success
```

Reply:
Alternatively:

```suggestion
   As ``PyStructSequence_InitType``, but return ``0`` on success
```

... or:

```suggestion
   Like ``PyStructSequence_InitType``, but return ``0`` on success
```

Reply:
I think in this context `returns` is correct: "the same as funcion, but (function) returns"